migration/mig_0016_add_recommendations_table.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright © 2020 Red Hat, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migration
+
+import (
+	"database/sql"
+	"encoding/json"
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+var mig0016AddRecommendationsTable = Migration{
+	StepUp: func(tx *sql.Tx, driver types.DBDriver) error {
+		_, err := tx.Exec(`
+		CREATE TABLE recommendations (
+			cluster_id      VARCHAR NOT NULL,
+			rule_fqdn       VARCHAR NOT NULL,
+			error_key		VARCHAR NOT NULL,
+			PRIMARY KEY(cluster_id)
```

Comment:
does it mean that there will be just one record per cluster?

Reply:
True, true. I don't even remember why I changed it to just cluster_id :)

Reply:
:)